src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Calculator from './components/Calculator';
+// import Calculator from './components/Calculator';
```

Comment:
- [ ] Kindly remove commented code as this will make your codebase cleaner and more readable to other future collaborators.